astropy/visualization/scripts/tests/test_fits2bitmap.py

```diff
@@ -60,7 +60,7 @@ def test_orientation(self, tmpdir):
         Regression test to check the image vertical orientation/origin.
         """
 
-        filename = tmpdir.join(self.filename).strpath
+        filename = str(tmpdir.join(self.filename))
```

Comment:
This allows one to call tests from `ipython` without having a `tmpdir` (no idea where to import it from)

Reply:
LGTM

Reply:
@mhvk I just noticed that there are 5 other instances in this file also using `.strpath` (lines 23, 28, 33, 42, and 54).  Do those need to be changed also?

Reply:
A quick search reveals that there are a lot of tests across astropy that use tmpdir `.strpath`.  Is a larger PR needed?

Reply:
Yes, you're right, we should probably change all. It may be easiest if I remove this commit from the PR, since it is really not numpy-dev related.